src/accounts/ethereum.ts

```diff
@@ -40,11 +43,14 @@ export class ETHAccount extends Account {
      *
      * @param encryptedContent The encrypted content to decrypt.
      */
-    decrypt(encryptedContent: Buffer): Buffer {
+    async decrypt(encryptedContent: Buffer): Promise<Buffer> {
         if (this.wallet) {
             const secret = this.wallet.privateKey;
             return secp256k1_decrypt(secret, encryptedContent);
         }
+        if (this.provider) {
+            this.provider.decrypt(encryptedContent);
```

Comment:
There is no return here. Will this still throw the error line 54 ?

Reply:
I'm surprised Typescript did not catch that... but it is fixed.